src/cio.ts

```diff
@@ -115,24 +126,46 @@ export async function identifyUser({
     }),
   ]);
 
+  const getDate = (value: number | string | Date) => {
+    if (typeof value === 'number') {
+      return debeziumTimeToDate(dup.createdAt);
```

Comment:
Feels like a copy paste issue?

```suggestion
      return debeziumTimeToDate(value);
```

Also this seems like a generic function, so I don't think it needs to be here

Reply:
Let me move this to our utils file 👀